docs/docs/persistent-volumes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Stateful Applications Using Local Persistent Volumes
+---
+
+Marathon applications lose their state when they terminate and are relaunched. In some contexts, for instance, if your application uses MySQL, you’ll want your application to preserve its state. You can create a stateful application by specifying a local persistent volume.
+
+When you specify a local volume or volumes, tasks and their associated data are “pinned” to the node they are first launched on and will be relaunched on that node if they terminate. Local volumes enable stateful tasks because the task can be restarted without data loss.
```

Comment:
Add (possibly after `if they terminate.`): `Resources required to run this application instance are reserved in order to make sure that the application can actually be relaunched on a node.` or something similar. This background knowledge is imo important, not only for the following section.